tools/c7n_gcp/c7n_gcp/entry.py

```diff
@@ -17,6 +17,10 @@
 import c7n_gcp.policy
 
 # This is an alphabetically sorted import list
+import c7n_gcp.actions
+import c7n_gcp.filters
+import c7n_gcp.output
+import c7n_gcp.policy
```

Comment:
the policy one is redundant (imported higher up). fwiw/unrelated, but I've got some work in progress on lazy loading all resources which speeds up cli interactions.